design/standard/templates/content/datatype/edit/eztags.tpl

```diff
@@ -1,9 +1,10 @@
 {def $permission_array = $attribute.content.permission_array}
+{def $show_dropdown = $attribute.contentclass_attribute.data_int2}
 
 {default attribute_base=ContentObjectAttribute}
 <div id="eztags{$attribute.id}" class="tagssuggest{if $attribute.contentclass_attribute.data_int2} tagsfilter{/if}"
     data-eztags
-    {* data-builder="MultipleSelects" *}
+    {if $show_dropdown}data-builder="Select"{/if}
```

Comment:
Maybe here should be some kind of string to define builder. Because you could create 2-3 additional views for displaying tags.

eg.
```tpl
{if $custom_builder}data-builder="{{$custom_builder}}"{/if}
```